### PR TITLE
[TR] PIM-5017 : fix media migration with lots of files

### PR DESCRIPTION
**Pull request by jmleroux**
Fix media migration performance

### CODE REVIEW

File: upgrades/1.3-1.4/common/AbstractMediaMigration.php [hunk 1]
```diff
@@ -193,4 +199,9 @@ protected function bootKernel($env)
 
         return $kernel;
     }
+
+    protected function writeConsole($message)
+    {
+        $this->output->writeln('<info>' . date('Y-m-d H:i:s') . '- </info>' . $message);
```
Comment (ClementGautier) [inline upgrades/1.3-1.4/common/AbstractMediaMigration.php]: sprintf

Comment (ClementGautier) [inline upgrades/1.3-1.4/common/AbstractMediaMigration.php]: sprintf

**Comment by nidup** [comment]
GTM

**Comment by jjanvier** [comment]
GTM

**Pull request merged**